test/widgets/test_thermal_zone.py

```diff
@@ -0,0 +1,14 @@
+import os.path
+from libqtile import widget
+
+def test_thermal_zone_getting_value():
+    # If thermal zone zero exists - get value
+    zone_zero = '/sys/class/thermal/thermal_zone0/temp'
+    zone_exists = os.path.isfile(zone_zero)
+    
+    if zone_exists:
```

Comment:
This test only works if there is a `thermal_zone0` on the CI machine.

It would be better to mock the contents of that file so that you can check the widget parses the contents correctly.

You could also test the error handling in your code (once you've added it).

Reply:
Done